crowbar_framework/app/assets/javascripts/installer.js

```diff
@@ -0,0 +1,73 @@
+$(document).ready(function() {
+  $("body.installers").each(function() {
+    function statusCheck() {
+      $.ajax({
+        type: "GET",
+        dataType: "json",
+        url: Routes.status_installer_path(),
```

Comment:
Identifier 'status_installer_path' is not in camel case.<br>'Routes' is not defined.